src/lib/ColorSwatch/ColorSwatch.scss

```diff
@@ -1,36 +1,38 @@
-.color-swatch {
-	margin: 0;
-	appearance: none;
+:global {
```

Comment:
Is this needed now?

Reply:
Yes. Since the Fluent Svelte tooltip is added as a wrapper element, Svelte's CSS scoping prevents the `<input>` inside from being scoped directly.

Yes, this is one of the most controversial and contentious discussions in the Svelte community.